backend/app/notify/telegram.go

```diff
@@ -16,38 +16,44 @@ import (
 	"github.com/pkg/errors"
 )
 
+// TelegramParams contain settings for telegram notifications
+type TelegramParams struct {
```

Comment:
what the reason for adding this? 3-4 params directly in the constuctor looks fine to me

Reply:
Hiding API, which is used only for tests, and adding UserNotifications param in PR for a telegram user notifications, which will raise the count of params by one: I felt silly adding the new param to all the tests which even don't require it, five params to function felt like too many.

Reply:
I didn't mean passing that apiPrefix via constructor but rather setting it directly in tests. anyway, it doesn't bother me enough, so fine with me ;)